src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -289,26 +291,25 @@ public void ErrorListInfoBarController_CurrentBackgroundProcessorCancellation()
         }
 
         [TestMethod]
-        public void ErrorListInfoBarController_Refresh_NonCriticalExceptionIsSuppressed()
+        public async Task ErrorListInfoBarController_Refresh_NonCriticalExceptionIsSuppressed()
         {
-            var logger = new TestLogger();
             var host = CreateHostWithThrowingConfigProvider(new COMException("thrown by test code"));
-            var testSubject = new ErrorListInfoBarController(host, Mock.Of<IUnboundProjectFinder>(), logger);
+            var testSubject = CreateTestSubject(host);
```

Comment:
Miscellaneous: most of the test were using the `CreateTestSubject` method but not all. Now they all do.